andela/titan-dev-ah-frontend
# #165441605 Chore Current User

#### What does this PR do?
Set current use to be accessible anywhere in the app
#### How should this be manually tested?
- clone the repo and check out the following branch `ft-set-current-user-165441605 `
- run `yarn start` and navigate to /sign_in in your browser
- And the current user will be accessible from the redux state - auth
### Related Pivotal tracker ID
#165441605
